library/core/src/primitive_docs.rs

```diff
@@ -1181,6 +1184,9 @@ mod prim_f64 {}
 /// as many bits as `f64`. Please see [the documentation for [`prim@f32`] or [Wikipedia on
 /// quad-precision values][wikipedia] for more information.
 ///
+/// Note that not all major platforms have hardware support for f128, in which case a
+/// software implementation will be used. This may be slower than expected.
+///
```

Comment:
The wording can be a bit stronger since performance really is not great.

```suggestion
/// Note that most major platforms do not have hardware support for `f128`, in which case a
/// software implementation will be used. This can be significantly slower than using `f64`.
///
```